2022q3/gcc.adoc

```diff
@@ -0,0 +1,31 @@
+=== GCC: New maintainer, GCC 12.2 and more
+
+Links: +
+link:https://gcc.gnu.org[GCC Project] URL: link:https://gcc.gnu.org[https://gcc.gnu.org] +
+link:https://gcc.gnu.org/gcc-11/[GCC 11 release series] URL: link:https://gcc.gnu.org/gcc-11/[https://gcc.gnu.org/gcc-11/]
+link:https://gcc.gnu.org/gcc-12/[GCC 12 release series] URL: link:https://gcc.gnu.org/gcc-12/[https://gcc.gnu.org/gcc-12/]
+
+Contact: <toolchain@FreeBSD.org> +
+Contact: Lorenzo Salvadore <salvadore@FreeBSD.org> +
+
+ * salvadore@ adopted all existing ports corresponding to supported versions of gcc, namely: lang/gcc10, lang/gcc11, lang/gcc11-devel, lang/gcc12, lang/gcc12-devel and lang/gcc13-devel.
+   At the moment -devel ports are updated weekly, unless a build failure makes it impossible.
+   Of course, in the latter case, the build failure is fixed and/or reported upstream as soon as possible.
+
+ * link:https://gcc.gnu.org/gcc-12/changes.html[GCC 12.2 has been released] and thus work has started to update the default GCC version to version 12.
+   Thank you very much to antoine@ who has already run the firt exp-run and to all the contributors, maintainers and committers involved in the process.
+   link:https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=265948[https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=2659548]
+
+ * Discussion about LTO keep going with many different point of views.
+   If interested, you can read the last contributions to the topic link:https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=264949[here] and link:https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=265254[here].
```

Comment:
Rewriting to
>    If interested, you can read the latest contributions to the topic: link:https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=264949[lang/gcc11: Needs build time warning for /tmp consumption] and link:https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=265254[lang/gcc11: build gets stuck].

s/last/latest/ (the most recent contributions, not the last contributions ever)
Also changed the link texts for accessibility: see https://webaim.org/techniques/hypertext/link_text for background.